prog/dftb+/prg_dftb/mainio.F90

```diff
@@ -2193,7 +2193,8 @@ subroutine writeBandOut(fd, fileName, eigen, filling, kWeight)
       do iK = 1, size(eigen, dim=2)
         write(fd, *) 'KPT ', iK, ' SPIN ', iSpin, ' KWEIGHT ', kWeight(iK)
         do iEgy = 1, size(eigen, dim=1)
-          write(fd, "(2f12.5)") Hartree__eV * eigen(iEgy, iK, iSpin), filling(iEgy, iK, iSpin)
+          write(fd, "(I6,2f12.5)") iEgy, Hartree__eV * eigen(iEgy, iK, iSpin),&
+              & filling(iEgy, iK, iSpin)
```

Comment:
Note: the tool `dp_bands` must be adapted to be able to cope with this.

Reply:
Probably should make this backward compatible to the old format. Can you have a look at the regex in  dptools/bandout.py for this one?